sky/engine/core/events/GestureVelocity.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/engine/core/events/GestureVelocity.h"
+
+namespace blink {
+
+GestureVelocity::GestureVelocity(bool valid, float vx, float vy)
```

Comment:
I had it that way originally but I ended up with expressions like velocity.velocityX which seemed too verbose.